### ISSUE TITLE
Invalid ntopng connections

### ISSUE THREAD

**Issue by lucaderi** [ntop/ntopng#5730]
ntopng.exe , downloaded and installed from ntopng-4.3.210218-x64.zip  makes the following outgoing network connections.

processPath    proto    remoteIP    remotePort
C:\Program Files\ntopng\ntopng.exe    udp    81.164.131.245    5353
C:\Program Files\ntopng\ntopng.exe    udp    81.164.131.245    5353
C:\Program Files\ntopng\ntopng.exe    tcp    81.164.131.245    80
C:\Program Files\ntopng\ntopng.exe    tcp    81.164.131.245    80
C:\Program Files\ntopng\ntopng.exe    tcp    81.164.131.245    80
C:\Program Files\ntopng\ntopng.exe    tcp    81.164.131.245    80
C:\Program Files\ntopng\ntopng.exe    udp    212.53.163.30    5353
C:\Program Files\ntopng\ntopng.exe    tcp    212.53.163.30    80
C:\Program Files\ntopng\ntopng.exe    tcp    212.53.163.30    80
C:\Program Files\ntopng\ntopng.exe    udp    203.228.2.161    5353
C:\Program Files\ntopng\ntopng.exe    tcp    203.228.2.161    80
C:\Program Files\ntopng\ntopng.exe    tcp    203.228.2.161    80

https://packages.ntop.org/Windows/old/ntopng-4.3.210218-x64.zip

**Comment by simonemainardi** [comment]
I went through the code base to find the reason of these spurious requests. I believe the reason is due to the `NetworkDiscovery`.

The sequence of events can be summarized as follow.

1. A subnet of up to /23 is probed to scan hosts within the interface network

https://github.com/ntop/ntopng/blob/d7edcfeaf23ebea623f4c256b124d86c848ef449/src/NetworkDiscovery.cpp#L158

2. Those hosts are probed for MDNS and this causes the first request to 5353:

https://github.com/ntop/ntopng/blob/d7edcfeaf23ebea623f4c256b124d86c848ef449/src/NetworkDiscovery.cpp#L270

3. Then, SSDP discovery is performed to 293.255.255.250. Hosts in the LAN respond and responses are dissected to extract the `Location:`

https://github.com/ntop/ntopng/blob/d7edcfeaf23ebea623f4c256b124d86c848ef449/src/NetworkDiscovery.cpp#L873

4. Extracted location is then requested via HTTP to extract additional device information and this causes the subsequent request to port 80.

https://github.com/ntop/ntopng/blob/d7edcfeaf23ebea623f4c256b124d86c848ef449/scripts/lua/modules/discover_utils.lua#L1039


----

NOTES:

- I am not sure why hosts are probed using their unicast addresses in 2.
- Although security checks are enforced in 4. to avoid SSRF and XSS, the HTTP request is still output.

In conclusion, ss the ntopng of this issue is running on a AWS cloud environment, I believe there is L2 visibility of other hosts that are suspicious as those above.

**Comment by lucaderi** [comment]
Working as described. Will reopen if necessary
